spot-ml/src/main/scala/org/apache/spot/netflow/FlowSuspiciousConnectsAnalysis.scala

```diff
@@ -61,39 +68,18 @@ object FlowSuspiciousConnectsAnalysis {
     val invalidFlowRecords = filterAndSelectInvalidFlowRecords(inputFlowRecords)
     dataValidation.showAndSaveInvalidRecords(invalidFlowRecords, config.hdfsScoredConnect, logger)
 
+    outputFlowRecords
```

Comment:
This change looks very similar to the one being requested in SPOT-128 but feels a litter incomplete. 
The code base for this function is originally returning Unit. This change seems to serve only to TestFlow script, do we want to make this change and then complete the desired functionality in SPOT-128 or remove this change and let SPOT-128 take care?